flow360/component/simulation/framework/updater.py

```diff
@@ -15,8 +15,33 @@ def _no_update(params_as_dict):
     return params_as_dict
 
 
+def _24_11_0_to_24_11_1_update(params_as_dict):
+    # Check and remove the 'meshing' node if conditions are met
+    if "meshing" in params_as_dict:
+        meshing_defaults = params_as_dict["meshing"].get("defaults", {})
```

Comment:
This will not work for a case starting from VolumeMesh where `params_as_dict["meshing"] = None`
```suggestion
    if params_as_dict.get("meshing", None):
```

Reply:
done in: https://github.com/flexcompute/Flow360/pull/559/commits/170ff71fa8ff280d50cc9c9eba82d761bd831486